### DISCUSSION TITLE
希望能增加一键返回主页面的功能

### DISCUSSION THREAD

**Discussion by cqjerry** [zyronon/V2Next#11] [Ideas]
脚本很赞，好评！
希望能增加一键返回初始主页面的功能，
因为有时帖子列表看着看着就很下面了，回到最顶端挺麻烦的。

**Comment by zyronon** [comment]
你要不试试这个 手势插件？https://chrome.google.com/webstore/detail/smartup-gestures/bgjfekefhjemchdeigphccilhncnjldn?utm_source=chrome-ntp-icon。按住鼠标右键往上滑就到顶部了
